guide/interactions/buttons.md

```diff
@@ -0,0 +1,321 @@
+You can also send message components within an ephemeral response or alongside message embeds.
+
+```js {1,13-19}
+const { MessageActionRow, MessageButton, MessageEmbed } = require('discord.js');
+
+client.on('interaction', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const row = new MessageActionRow()
+			.addComponents(new MessageButton()
+				.setCustomID('primary')
+				.setLabel('primary')
+				.setStyle('PRIMARY'));
+
+		const embed = new MessageEmbed()
+			.setColor('#0099ff')
+			.setTitle('Some title')
+			.setURL('https://discord.js.org/')
+			.setDescription('Some description here');
+
+		await interaction.reply({ content: 'Pong!', ephemeral: true, embeds: [embed], components: [row] });
+	}
+});
+```
+
+Restart your bot and then send the command to a channel your bot has access to. If all goes well, you should see something like this:
+
+<!--- vue-discord-message doesn't yet have support for inline replies/interactions/ephemeral messages/components -->
+<div is="discord-messages">
+	<discord-message profile="bot">
+		Pong! (ephemeral + components)
+		<discord-embed
+			slot="embeds"
+			color="#0099ff"
+			title="Some title"
+			url="https://discord.js.org/"
+		>
+			Some description here
+		</discord-embed>
+	</discord-message>
+</div>
+
+Additionally, if you don't want to construct an `ActionRow` every time, you can also pass an array of arrays containing components like this:
+
+```js {7-10,18}
+const { MessageActionRow, MessageButton, MessageEmbed } = require('discord.js');
+
+client.on('interaction', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const button = new MessageButton()
+			.setCustomID('primary')
+			.setLabel('primary')
+			.setStyle('PRIMARY');
+
+		const embed = new MessageEmbed()
+			.setColor('#0099ff')
+			.setTitle('Some title')
+			.setURL('https://discord.js.org/')
+			.setDescription('Some description here');
+
+		await interaction.reply({ content: 'Pong!', ephemeral: true, embeds: [embed], components: [[button]] });
+	}
+});
+```
+
+### Disabled buttons
+
+If you want to prevent a button from being used, but not remove it from the message, you can make use of the `setDisabled()` method, which will make it unclickable for everyone:
+
+```js {13}
+const { MessageActionRow, MessageButton } = require('discord.js');
+
+client.on('interaction', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const row = new MessageActionRow()
+			.addComponents(
+				new MessageButton()
+					.setCustomID('primary')
+					.setLabel('primary')
+					.setStyle('PRIMARY')
+					.setDisabled(true),
+			);
+
+		await interaction.reply({ content: 'Pong!', components: [row] });
+	}
+});
+```
+
+### Emoji buttons
+
+The label of a button can hold unicode emojis, but if you want to use guild emojis you have to use the `setEmoji()` method like so:
```

Comment:
Label emojis and explicit button emojis are not quite the same thing, detailing that unicode emojis can be part of the label is warranted, but you can also still prefix the button label with a button emoji (different places of availability inside the interaction payload, unless that changed very recently)